build-tools/packages/version-tools/package.json

```diff
@@ -20,7 +21,7 @@
 		}
 	},
 	"main": "lib/index.js",
-	"types": "lib/index.d.ts",
+	"types": "./lib/index.d.ts",
```

Comment:
I think I'd go without the leading dot but don't mind too much to be honest. However, whichever way, can we make it match the way "main" is specified?

Also worth noting that build-cli also acquired `type: commonjs` but didn't have its `types` line changed...?

Reply:
> Also worth noting that build-cli also acquired `type: commonjs` but didn't have its `types` line changed...?

These were manually added. I'll update the PR description.

Reply:
> I think I'd go without the leading dot but don't mind too much to be honest. However, whichever way, can we make it match the way "main" is specified?

The latest version now normalizes the main and types fields to match exports.